plugins/omemo/src/ui/manage_key_dialog.vala

```diff
@@ -58,7 +58,7 @@ public class ManageKeyDialog : Gtk.Dialog {
         verify_yes_button.clicked.connect(() => {
             confirm_image.set_from_icon_name("security-high-symbolic", IconSize.DIALOG);
             confirm_title_label.label = _("Verify key");
-            confirm_desc_label.set_markup(_("Once confirmed, any future messages sent by %s using this key will be highlighted accordingly in the chat window.").printf(@"<b>$(device[db.identity_meta.address_name])</b>"));
```

Comment:
"Once confirmed" is redundant. It should be clear that this describes what happens when the user confirms